preferences/src/test/kotlin/ktx/preferences/preferencesTest.kt

```diff
@@ -0,0 +1,118 @@
+package ktx.preferences
+
+import com.badlogic.gdx.Preferences
+import com.badlogic.gdx.utils.ObjectMap
+import com.nhaarman.mockitokotlin2.any
+import com.nhaarman.mockitokotlin2.times
+import com.nhaarman.mockitokotlin2.verify
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+import org.mockito.Mockito
+
+class PreferencesTest {
+  private lateinit var preferences: Preferences
+  private val map = ObjectMap<String, Any>()
+
+  @Before
+  fun `setup preferences`() {
+    preferences = Mockito.mock(Preferences::class.java)
```

Comment:
Since you mock (nearly?) the entire class, you might as well implement `Preferences` in tests. Should be easier to read than a bunch of mocks.